examples/on_saved/pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[build-system]
+requires = ["setuptools>=45", "setuptools-scm[toml]>=7.1"]
+build-backend = "setuptools.build_meta"
+
```

Comment:
I'm not sure about the practice to check in pyproject and requirements files for code examples anymore.

Could we instead make a note in the accompanying document that dependencies abc and xyz need to be installed? These files are largely useless for a reader of the docs that just follows along on his/her machine.

Reply:
@janwillemkl remarked on the Streamlit integration to prepare a pyproject file for the example. That is why I prepared one here. 

It is essentially a `pip install datasets transformers[torch] nnbench`, so it would also work without a requirements file, I guess.